LuaRules/Gadgets/unit_decloak_damaged.lua

```diff
@@ -320,6 +321,14 @@ function gadget:UnitCreated(unitID, unitDefID)
 end
 
 function gadget:Initialize()
+	local cp
+	for i=1, #WeaponDefs do -- cache pure slow and disarm damage weapons
+		cp = WeaponDefs[i].customParams or {}
```

Comment:
customparams is never nil
```suggestion
		cp = WeaponDefs[i].customParams
```